pkg/helper/machinepools/helpers_test.go

```diff
@@ -330,57 +330,3 @@ var _ = Describe("Validate node drain grace period", func() {
 		),
 	)
 })
-
-var _ = Describe("Validate MaxSurge and MaxUnavailable", func() {
```

Comment:
@hunterkepley I don't see this test being replicated anywhere. It looks like it's just been deleted?